_pytest/test_sanity.py

```diff
@@ -21,11 +21,11 @@ def import_submodules(package_name, skip_list):
     for loader, name, is_pkg in pkgutil.walk_packages(package.__path__):
         full_name = package.__name__ + '.' + name
         if full_name not in skip_list:
-            module = importlib.import_module(full_name)
-            if PY2:
-                reload(module)
-            else:
-                importlib.reload(module)
+            imported_module = importlib.import_module(full_name)
+            # if PY2:
```

Comment:
Ok, feel free to remove the test. It is brittle anyway.

Maybe we can do it differently, e.g. run a separate test suite within travis that only has necessary dependencies installed.